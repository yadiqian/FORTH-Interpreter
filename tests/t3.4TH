9 4 / 3 2 - ^ 5 8 4 * DUP .