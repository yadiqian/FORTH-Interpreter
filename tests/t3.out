32
Stack is not empty
[Integer 32,Integer 5,Integer 2]